src/plugins/loan-channel-query-map.js

```diff
@@ -397,6 +397,29 @@ export default {
 					queryParams: 'status=fundRaising&isGroup=1&distributionModel=both',
 				},
 
+				// Lend By Category Carousel Layout Sub Channels
+				// 	ACK-357 Experiment
+				{
+					id: 116,
+					url: 'solar-energy',
+					queryParams: 'queryString=solar&tag=9&sortBy=popularity'
+				},
+				{
+					id: 117,
+					url: 'sustainable-agriculture',
+					queryParams: 'sector=1&tag=8&sortBy=amountLeft'
+				},
+				{
+					id: 118,
+					url: 'recycle-and-re-use',
+					queryParams: 'queryString=used clothing&distributionModel=field_partner&sortBy=popularity',
+				},
+				{
+					id: 119,
+					url: 'other-eco-friendly-loans',
+					queryParams: 'tag=9,8&distributionModel=field_partner&sortBy=expiringSoon',
```

Comment:
shouldn't `tag` be `loanTags` ?

Reply:
I don't think so -- here is the link for solar: https://www.kiva.org/lend?tag=9&queryString=solar&sortBy=popularity

Reply:
ok so long as it's working properly in `/lend` it should be ok. Could you please also add this to the excludeParams

Reply:
Ok so both `tag` and `loanTags` works 🤷